shared/oae/api/oae.api.authentication.js

```diff
@@ -237,7 +237,11 @@ define(['exports', 'jquery', 'oae.api.config', 'oae.api.i18n', 'oae.api.user', '
 
         // Use the `auth.html` `authExternalButton` macro to create a form that performs this
         // authentication
-        var $template = $('<div class="hide">${authExternalButton(strategy, opts)}</div>');
+        var $template = $('<div><!--'
+                          +   '<div class="hide">'
```

Comment:
Maybe we can give this `div` an `id` so we can use that to limit the scope of the `$('.btn-external-auth')` selector below.